modules/references/client/components/read-experiences/ExperienceCounts.js

```diff
@@ -1,38 +1,213 @@
-  ).length;
-  const negativeCount = publicExperiences.filter(
+
+  const nonRecommendCount = experiences.filter(
     ({ recommend }) => recommend === 'no',
   ).length;
 
+  // One sentence summary about experiences.
+  const renderSummarySentece = () => {
+    let summary;
+
+    if (totalCount === 1 && recommendCount === 1) {
+      summary = t(
+        'One member shared their experience and they recommended them.',
+      );
+    } else if (totalCount === 1 && nonRecommendCount === 1) {
+      summary = t(
+        'One member shared their experience and they would not recommend them.',
+      );
+    } else {
+      summary = t('{{count}} members have shared their experiences.', {
+        count: totalCount,
+      });
+    }
+
+    return <SummarySentence className="lead">{summary}</SummarySentence>;
+  };
+
+  // Details about recommendations.
+  const renderRecommendationStats = () => {
+    const summaries = [];
+
+    if (totalCount === recommendCount) {
+      summaries.push(t('Everyone recommends them.'));
+    }
+
+    if (totalCount === nonRecommendCount) {
+      summaries.push(t('Everyone said they would not recommend them.'));
+    }
+
+    if (totalCount !== recommendCount && nonRecommendCount > 0) {
+      summaries.push(
+        t('{{count}} did not recommend.', {
+          count: nonRecommendCount,
+        }),
+      );
+    }
+
+    if (totalCount !== recommendCount && recommendCount > 0) {
+      summaries.push(
+        t('{{count}} recommended them.', {
+          count: recommendCount,
+        }),
+      );
+    }
+
+    return summaries.length > 0 ? (
+      <p>
+        <SummaryIcon icon={nonRecommendCount > 0 ? 'close' : 'ok'} />
+        {summaries.join(' ')}
+      </p>
+    ) : null;
+  };
+
+  // Statistics about "met", "host", "guest" interactions.
+  const renderInteractionStats = () => {
+    const interactions = [];
+
+    const getInteractionPercentage = interaction => {
+      const count = experiences.filter(({ interactions }) =>
+        Boolean(interactions[interaction]),
+      ).length;
+
+      return parseInt((count / totalCount) * 100, 10);
+    };
+
+    const met = getInteractionPercentage('met');
+    const hostedMe = getInteractionPercentage('hostedMe');
+    const hostedThem = getInteractionPercentage('hostedThem');
+
+    if (hostedThem === 100) {
+      interactions.push(t('They hosted everyone.'));
+    } else if (hostedThem > 0) {
+      interactions.push(
+        t('They hosted {{percentage}}% of members.', {
+          percentage: hostedThem,
+        }),
+      );
+    }
+
+    if (hostedMe === 100) {
+      interactions.push(t('Was hosted by everyone.'));
+    } else if (hostedMe > 0) {
+      interactions.push(
+        t('Was hosted by {{percentage}}% of members.', {
+          percentage: hostedMe,
+        }),
+      );
+    }
+
+    if (met === 100) {
+      interactions.push(t('Met with everyone.'));
+    } else if (met > 0) {
```

Comment:
What if the user was shared experiences only by those they hadn't met? Don't we want to have it in the summary?

Reply:
Good point!